lib/watir/elements/input.rb

```diff
@@ -0,0 +1,14 @@
+module Watir
+  class Input < HTMLElement
+
+    #
+    # Returns applicable label.
+    # Is not lazy loaded.
+    #
+
+    def label
+      parent(tag_name: 'form').label(for: id)
```

Comment:
Huh, interesting. I went to form because RadioSet doesn't make sense to me outside of a form context. These are independent of each other:
```html
<form name="sign_in" class="user">
    <input type="radio" name="opt">Opt In</input>
    <input type="radio" name="opt">Opt Out</input>
</form>
<form name="sign_up" class="user">
  <input type="radio" name="opt">Opt In</input>
  <input type="radio" name="opt">Opt Out</input>
</form>
```
If I went to `body` tag they would be considered the same `RadioSet` which they are not.